### PR TITLE
wrap: Focus floating windows only when visible.

### PR DESCRIPTION
**Pull request by sathya-pramodh**

### CODE REVIEW

File: waywall/wrap.c [hunk 1]
```diff
@@ -290,12 +290,13 @@ on_button(void *data, uint32_t button, bool pressed) {
     }
 
     // If a window grab should start, then start it. Otherwise, focus the window that the user
-    // clicked.
+    // clicked. (Only happens when the floating view is visible)
+    if (!wrap->floating.visible)
```
Comment (tesselslate) [inline waywall/wrap.c:294]: I try to use brackets everywhere, even if there is only a single statement in the block.

**Review by tesselslate — CHANGES_REQUESTED** [review]
LGTM, just want to use brackets everywhere

File: waywall/wrap.c (hunk 1 above)
Comment (sathya-pramodh) [reply waywall/wrap.c:294]: gotcha... ill push a commit for that one sec

**Comment by tesselslate** [comment]
Thanks! :+1:

**Pull request merged**